web/packages/teleport/src/Assist/Conversation/AccessRequests/TimelineItem.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+
+import { ApplicationsIcon, LeafIcon, ServerIcon } from 'design/SVGIcon';
+
+import { formatRelative } from 'date-fns';
+
+import type { AccessRequestEvent } from 'teleport/Assist/types';
+import { Resources } from 'teleport/Assist/Conversation/AccessRequests/Resources';
+
+interface TimelineItemProps {
+  event: AccessRequestEvent;
+}
+
+const Container = styled.div`
+  display: flex;
+  margin-left: 20px;
+`;
+
+const Timestamp = styled.div`
+  font-size: 12px;
+  position: absolute;
+  right: 0;
+  margin-left: 10px;
+  color: ${p => p.theme.colors.text.muted};
+`;
+
+const Description = styled.div`
+  font-size: 14px;
+  padding: 10px 0;
+`;
+
+const Header = styled.div`
+  display: flex;
+  position: relative;
+  align-items: center;
+  padding-right: 70px;
+
+  &:after {
+    content: '';
+    position: absolute;
+    width: 10px;
+    height: 10px;
+    border-radius: 50%;
+    background: ${p => p.theme.colors.warning.main};
+    top: 7px;
+    left: -20px;
+  }

```

Comment:
What does this part do? It might be good to document this, unless you plan to add stories later.